tests/integration/cairo1_compiler_bindings/print_test/simple/lib.cairo

```diff
@@ -0,0 +1,12 @@
+#[contract]
+mod SimpleContract {
```

Comment:
```suggestion
mod PrintingContract {
```

Would rename the folder accordingly.

Reply:
[naming](https://github.com/software-mansion/protostar/pull/1803/commits/8c651fda6d4627194766a8aa9a452fc3506e169f)